oidc-controller/api/core/oidc/tests/__mocks__.py

```diff
@@ -1341,6 +1341,15 @@
 ver_config = VerificationConfig(
     ver_config_id="verified-email",
     subject_identifier="email",
+    metadata={
+        "title": {"en": "Get Name"},
+        "claims": {
+            "en": [
+                "That you are a BC Resident",
+                "That you are a member of the Law Society of British Columbia",
+            ]
+        },
+    },
```

Comment:
What about using a single language key for each entry, rather than duplicating it?
```suggestion
    metadata={
        "en":{
          "title": "Get Name",
          "claims": [
                "That you are a BC Resident",
                "That you are a member of the Law Society of British Columbia",
          ]
        }
    },
```